### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by Wusinho**
HTML & CSS3 : Mint-SignUp Project
- Used Semantics to improve HTML format.
- Used Form tag (input, submit, and labels)
- Linters and Validations applied to the code.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,180 @@
+body {
+  margin: 10px 30px;
+  font-family: inherit;
+  background-color: #eceef1;
+}
```
Comment (Temmarie) [inline css/style.css:5]: Please consider using "Sans-serif" as the font-family in the body, that way you don't have to specify a different font for each section.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,180 @@
+body {
+  margin: 10px 30px;
+  font-family: inherit;
```
Comment (Temmarie) [inline css/style.css]: Please consider using a "Sans-serif" font-family in the body, that way you don't have to specify a different font for each section of the page.
`body {
font-family: sans-serif;
}`

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle: 
### Hello Team! 
 
### You have done well so far :clap: However, there are some changes you have to make for your project to be approved. Kindly work on them and submit them for another code review.

 **General**



- [x] Your project looks great. However, your project cannot be approved until you write a descriptive Pull Request Message. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.

- [x] Please edit your Readme to contain a brief description about the Mint form clone. You currently have a description about the Youtube clone.

#### OPTIONAL
- [x] Try to implement the use of font awesome icons for the sign in button as well as future projects.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review @tamara._ :wink: 
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: css/style.css (hunk 2 above)
Comment (Wusinho) [reply css/style.css]: I have made the changes you suggested.

**Review by Temmarie — APPROVED** [review]
## Status: Approved :heavy_check_mark:

### Hello Team! 

### Thank you for making all of the required changes, your project is approved :clap: :clap: :clap: :tada:

### You may merge, good luck with your next projects! :rocket:
### Happy Coding! :+1:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Pull request merged**